internal/auth_request/model/request.go

```diff
@@ -19,10 +19,11 @@ const (
 )
 
 const (
-	OrgDomainPrimaryScope = "urn:zitadel:iam:org:domain:primary:"
-	OrgDomainPrimaryClaim = "urn:zitadel:iam:org:domain:primary"
-	ProjectIDScope        = "urn:zitadel:iam:org:project:id:"
-	AudSuffix             = ":aud"
+	OrgDomainPrimaryScope   = "urn:zitadel:iam:org:domain:primary:"
+	OrgDomainPrimaryClaim   = "urn:zitadel:iam:org:domain:primary"
+	OrgPrivateLabelingOrgID = "urn:zitadel:iam:org:private:labeling:id:"
+	ProjectIDScope          = "urn:zitadel:iam:org:project:id:"
+	AudSuffix               = ":aud"
```

Comment:
are they still needed? there are exactly the same values in the domain package

Reply:
I did it like with the primary domain scope. i don't know why we have it in two packages. but at the moment both are in use